src/main.cpp

```diff
@@ -0,0 +1,19 @@
+#include "zed_cpu.hpp"
+
+int main(int argc, char * argv[])
+{
+  ros::init(argc, argv, "zed_camera");
+
+  auto nh = std::make_shared<ros::NodeHandle>("~");
```

Comment:
```suggestion
  auto nh = std::make_shared<ros::NodeHandle>();
```
I don't see a point in making this a private nh

Reply:
Keeping similarity of topics to official zed-ros-wrapper